<!DOCTYPE HTML>
<!--
	LimsLab @ NKU
-->

<html>
	<head>
		<title>LimsLab @ NKU</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		
		<script>
			var _hmt = _hmt || [];
			(function() {
				var hm = document.createElement("script");
				hm.src = "https://hm.baidu.com/hm.js?637167ccc21fbb404ed76bb140e218c0";
				var s = document.getElementsByTagName("script")[0];
				s.parentNode.insertBefore(hm, s);
			})();
		</script>
		
	</head>
	<body class="homepage is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<section id="header">
					<div class="container">

						<!-- Logo -->
							<h1 id="logo"><a href="index.html">L<font color='blue'>ims</font>Lab <font color='black'>@ NKU</font></a></h1>
							<p><font color='black'><strong>Native Structural Mass Spectrometry</strong></font></p>
							<font color='light gray'>Towards Living Single-cell Protein Structures</font>

						<!-- Nav -->
							<nav id="nav">
								<ul>
									<li><a class="icon solid fa-home" href="index.html"><span>Home</span></a></li>
									<li><a class="icon fa-chart-bar" href="people.html" ><span>People</span></a>
										<ul>
											<li><a href="index.html#PI">PI</a></li>
											<li><a href="people.html#Team">Current Members</a></li>
											<li><a href="people.html#Alumni">Alumni</a></li>
											<li><a href="people.html#Collaborator">Collaborators</a></li>
										</ul>
									</li>
									<li><a class="icon solid fa-cog" href="news.html"><span>News</span></a></li>
									<li><a class="icon solid fa-retweet" href="gallery.html"><span>Gallery</span></a></li>
									<li><a class="icon solid fa-sitemap" href="research.html"><span>Research</span></a></li>
									<li><a class="icon solid fa-sitemap" href="publication.html"><span>Publication</span></a></li>
								</ul>
							</nav>

					</div>
				</section>

			<!-- Main -->
				<section id="main">
					<div class="container">
						<div id="content">

							<!-- Post -->
								<article class="box post">
									<header>
										<h2 class='major-header'><strong><a name="SMS" >"More Native" Structural Mass Spectrometry</a>:</strong> </h2>
										<h2 class='major-header'><strong><font color='red'>Rapid</font> + <font color='green'>Lossless</font> + <font color='blue'>High-resolution</font> +++ </strong> </h2>
										
									</header>
					
									<a class='image centered'> <img src="images/limslab_projects_full.jpg" alt="" style="width:60%;"/></a>
									<h3><U><strong><font color='red'>Rapid: Nanosecond Photochemical Structure Probing</font></strong></U></h3>
									<p>Dr. Li has the long-standing research interest focusing on the development 
									and application of <strong>ultrafast</strong> separation techniques capable of in situ purification 
									and identification of native proteins directly from living cells. As such, one needs 
									to in situ remove intracellular and extracellular matrices surrounding cell-trapped 
									target proteins, where the separation time scale should be as short as possible to 
									maintain the biologically relevant structural information. During his PhD career, 
									Gongyu has developed a ultrafast in-solution separation method and integrated into MS (Li et al. Anal. Chem. 2016, 88, 10860-10866), 
									and has demonstrated its utility for several protein systems directly from living cells 
									(Li et al. Anal. Chem. 2018, 90, 3409-3415). In this manner Dr. Li has assembled both ultrafast separation and online electroporation-induced protein release into a 
									small region inside a nanospray needle. In a recent study (Anal. Chem. 2019, 91, 10441-10447), 
									Dr. Li and coworkers have provided further evidence of ultrafast separation. Meanwhile, Dr. Li’s ultrafast separation work 
									facilitated the mechanistic investigation of click chemical reactions (Chem. Sci. 2017, 8, 214-222). Benefiting from Gongyu’s contribution to 
									improved understanding of the basic mechanisms and early applications of ultrafast separation, 
									this separation technique has recently been used for in situ chemical profiling of single 
									living neurons (PNAS, 2018, 114, 2586-2591) and for revealing a novel glutamate biosynthetic 
									pathway in the brain (Cell, 2018, 173, 1716-1727). In addition to contribution to the 
									solution-phase separation improvement, Gongyu also devoted himself in the development of online gas-phase structural 
									technique, such as the development of a new regime for <strong>ion mobility-mass spectrometry</strong> 
									(IM-MS)-based identification of soltuion-phase conformational changes (Li et al. Anal. Chem. 2018, 90, 7997-8001). 
									Dr. Li is ambitious to combine ultrafast solution-phase separation with gas-phase structural MS tool, 
									IM-MS, serving as an emerging alternative to probe more structural details of native proteins 
									directly from living cells.</p>
									<p>While Dr. Li is much interested in serving as human disease fighter, he believes the high-quality 
									technological and analytical developments are of most fundamental and prerequisite importance before 
									biologist and us can take any further actions facing cutting-edge challenges. <strong>Photochemical reactions (PCRs)</strong> 
									have long been widely involved in many aspects from fundamental research to industry-wide applications. 
									In another paper (Li et al. Nat. Commun. 2019, 10, 4697, FEATURED by Nature Communications Editors HIGHLIGHTS), 
									he demonstrated the suitability and possibility to adapt <strong>nanosecond PCR (nsPCR)</strong> with mass spectrometry (MS) 
									for multiple analytical measurements and biological applications. This development has enabled sample separation, 
									MS ionization, and chemical reaction to be integrated into a localized space and achieved in a few nanosecond 
									time scale. </p>
									<p>Simultaneous protein identifications and their structural probing are frequently beset with tedious, 
									multistep and structural perturbing sample preparations. The resultant relatively independent workflows 
									for each other have thus discouraged their applications in rapid and reliable probing of biological and 
									medical events. In this work, Dr. Li presented a “Two-Birds-With-One-Stone” photochemical strategy for a 
									trade-off between large protein structural probing and on-demand matrix removal. The nsPCR is designed to 
									enable simultaneous high-throughput, highly efficient protein labeling and on-demand matrix removal at nanosecond 
									timescale and micrometer confined environment. The labeling occurs between the nsPCR product, 2-nitrosobenzoic anion (NS-), 
									and primary amine groups at the N-terminus and lysine residues of proteins. The on-demand matrices removal benefits from the 
									composite effects of local pH-jump effect, photochemical microscale electrophoresis and microscale thermophoresis. 
									The labeling efficiency for various sample types has been demonstrated to be as high as 90% with a high-throughput feature. 
									As a proof-of-concept demonstration, the nsPCR has also revealed the stabilizing effects of terminal sialylation on glycoprotein 
									3D structures for the first time, which offers unique features and capabilities that cannot be readily achieved using a typical 
									gas-phase structural analysis tool, ion mobility-mass spectrometry.</p>
									<p>The key idea of nsPCR stemmed from previous use of photoactive compound 2-nitrobenzaldehyde (NBA), which releases 
									a proton upon UV laser irradiation (J. Am. Chem. Soc. 2016, 138, 5363), and generates amine reactive species. 
									Inspired by this JACS paper, Dr. Li began to explore its utility in matrix-assisted desorption ionization (MALDI)-MS 
									in terms of alleviating ion suppression effects. Notably, Dr. Li has been long interested in improving methods for 
									in situ identification and visualization of neuropeptides and proteins directly from multiple tissue samples, which 
									is often plagued by ion suppression effects. Based on the past experience on ion suppression effects in both ESI 
									(J. Mass Spectrom. 2014, 49, 639) and MALDI systems (Rapid Commun. Mass Spectrom. 2019, 33, 327), Dr. Li was excited 
									to observe signal enhancements with the help of NBA modification for numerous biomolecules, initially focusing on 
									neuropeptides and large proteins. Aside from signal improvement, a more intriguing observation was the detection of 
									several new peaks generated from the mixed MALDI-MS cocktail. Further examination revealed that these additional peaks 
									were due to specific NBA labeling of primary amine groups at the N terminus and lysine residues.</p>
									<p>We envision many more applications will be easily established based on the unique feature of NBA-based nsPCR, and 
									further development of more suitable systems to initiate nsPCR will greatly expand its general applicability as a 
									rapid and high-throughput structural probe. Taken together, these features of nsPCR, coupling with different MS 
									instrumentation platforms, would enable a broad range of biological applications that could benefit from in situ and 
									rapid structural probing of various biomolecules from diverse set of samples, including single cells and comparative 
									biological tissues from healthy and disease conditions.</p>
									
									<h3><U><strong><font color='green'>Lossless: AIU/AIF IM-MS</font></strong></U></h3>									
									<p>MS-based composite strategies have considerably dominated gas-phase glycoprotein 3D structural analysis due to 
									their inherent and indispensable capability in resolving glycan and glycoprotein microheterogeneity, which has 
									become even more versatile along with the continuous developments in native MS, tandem MS including collision-induced 
									unfolding (CIU), and ion mobility interfaces. Native ion mobility-mass spectrometry (IM-MS) has emerged as a top 
									choice for protein structure study at the accurate molecular-mass scale, especially with the recent remarkable 
									advancements in instrument resolving power, protein activation and manipulation techniques, and multiple data 
									processing and visualization methods. Notably, charge state selection has been widely acknowledged as a determinant 
									consideration to obtain reliable and biologically relevant structural information. Multiple independent research lines 
									have reported the common observation of charge state-dependent structures and topologies of nanospray generated protein ions.
									The unique gas-phase charge-state-distribution phenomenon that is distinct from  native solution phase, not only 
									adds uncertainties of the structural gaps between these two phases, but also increases the time cost and degrades 
									the confidence of gas-phase structural measurements.</p>
									<p>To effectively and confidently dissect the PTM effects on the overall 3D arrangements of glycoproteins 
									with high-level microheterogeneity, we introduce Native IM-based <strong>Lossless Structural MS (LSMS)</strong> strategy. 
									To overcome the barrier of multistep, time-consuming 
									charge-selection/optimization processes and charge-bias effects on protein structures, the LSMS strategy 
									adopts a “charge selection-free” concept, in which all forms of protein ions’ structures and topologies 
									are accounted for, and no precursor ion selection is mandatory. Namely, all ions are diverted into the activation cell. 
									In this way, we manage to perform <strong>all ion unfolding/fragmentation (AIU/AIF)</strong> sequentially in a charge-independent manner 
									on sialylated glycoproteins. The LSMS additionally relies on the further modifications of data visualization and 
									quantitative characterization based on the CIUSuite and UniDec. By using a model sialylated glycoprotein transferrin (TF), LSMS allows for not only the reliable tracking of rapid, 
									efficient and in situ releasing of sialic acids and the simultaneous, charge-independent stepwise unfolding of native-like 
									glycoproteins, but also unprecedentedly revealing the differential roles of Neu5Ac and Neu5Gc sialylations in fine-tuning 
									of glycoprotein structures and topologies. </p>
									<p>These results not only provide new molecular signatures for the preclinical functions of PTMs, but also 
									indicate that inhibitors that sustain Neu5Ac level at suitable values could be pursued as potential therapeutics to slow 
									the progression of several sialylated glycoprotein-linked diseases, including AD. Furthermore, we envision that the preclinical 
									implications and underexplored molecular basis of glycosylation other than terminal sialylations attainable using our LSMS platform 
									will find promising applications in other target glycoproteins in both health and disease conditions. It will be of great 
									interest to apply the current strategy to revealing molecular and structural basis for the association of the O-GlcNAcylation 
									of such glycoproteins with AD progression, as well as the linkage of the enhanced secretion of amyloid β peptides to abnormal 
									sialylation of amyloid β-precursor protein</p>
									
									<h3><U><strong><font color='blue'>High-resolution: Structural Amplification MS</font></strong></U></h3>	
									<p>Of concern is the recent failure of several clinical trials on AD, highlighting the urgent need for earlier, 
									possibly preventative intervention, and raising the question of what form of Aβ is the best target (Prion 2014, 
									8, 119). Chiral inversion of amino acids is thought to modulate the structure and function of many neuropeptides 
									(Science 1987, 238, 200/Biochem. Biophys. Res. Commun. 1997, 240, 354/ Proc. Natl. Acad. Sci. U. S. A. 2005, 102, 
									413/Bioanal. Rev. 2009, 1, 7/), including Aβ, but these processes are poorly understood. As chiral Aβ peptides with 
									partial amino acid D-isomerization have been previously detected in AD brain regions (J. Chromatogr. B 2011, 879, 
									3141/J. Neuropathol. Exp. Neurol. 2003, 62, 248), there is a possibility that D-isomerized Aβ play a vital role in 
									AD pathogenesis and development. However, since Aβ D-isomerization is slowly progressed, age-dependent and is present 
									at low stoichiometry (e.g. less than 10%, Neuroscience 2001, 104, 1003), the role of chiral Aβ has long been ignored 
									and largely underexplored, in part due to lack of effective tools. Recent studies (Nature 2018, 554, 249/J. Am. Soc. 
									Mass Spectrom. 2019, 30, 1325) further highlighted the long-term ignoration, and the importance, of stereoisomeric Aβ 
									isoforms, through pinpointing the isoform-dependent antibody selection during Aβ isolation and sampling from AD brains. 
									Towards this end, Dr. Gongyu Li present an innovative conception and potentially new drug target for Alzheimer’s disease 
									(AD) therapy by investigating chiral effects on Aβ peptide. In the paper published in Nature Communications (Nat. 
									Commun. 2019, 10, 5038), Dr. Li developed an integrated, multidimensional ion mobility-mass spectrometry (IM-MS)-based 
									approach to study chirality-regulated structural features of Aβ fragments and their influence on Aβ receptor recognition.</p>
									<p>Dr. Li developed and established an innovative analytical platform, benefiting from the rational designs that target 
									Aβ chiral chemistry (Figure 1). Consequently, distinct structural and molecular differences have been revealed between 
									wild type and D-isomerized Aβ, including its monomer structure, oligomerization behavior and its receptor-recognition 
									and binding characteristics. In addition to the crosstalking effects among those epimeric Aβ during oligomerization, the 
									differential contributions of the chirality of Aβ N-terminal and C-terminal fragments were also interrogated, suggesting 
									their inevitably cooperative effects. It is believed that current results could facilitate future investigations of novel 
									therapeutic treatments for AD as new insights can be obtained via elucidation of the roles of D-isomerized Aβ in early AD 
									development, diagnosis, and prognosis. Notably, while there are some previous efforts targeting to key interaction sites of 
									wild type Aβ through the design of D-amino acid peptide inhibitors (Prion 2014, 8, 119), our study just for the first time 
									highlighted the importance of considering key amino acid chirality in Aβ, when designing any types of inhibitors targeting 
									to disrupting or directing stereoisomeric Aβ to stable, nontoxic and off-pathway aggregation.</p>
									<p>To rationally design such stereoisomeric Aβ inhibitors, it is urgently required to better understand the differences 
									between stereoisomeric Aβ and wild type counterparts, in terms of many aspects including monomer structure, aggregation 
									pathway and receptor recognition behavior. The key idea of chiral amplification through metal binding was originally inspired 
									by previous reports including his own research experience on zinc finger peptide-zinc binding (J. Am. Soc. Mass Spectrom. 
									2017, 28, 2658) and other IM-MS-based peptide-metal binding studies (J. Am. Soc. Mass Spectrom. 2017, 28, 1293). These 
									previous studies suggested that metal binding is dependent on the structural features of peptide itself and can enhance 
									structural difference amongst peptide isomers. Dr. Li therefore selected copper as first candidate metal, as it has its 
									unique isotopic distribution that helps us identify the binding events much easier even only with MS1 measurements (e.g. 
									no MS/MS is mandatory). Besides, Cu2+ binds to most peptides with a moderate to high affinity and thus we can capture such 
									types of binding in the gas phase even after desolvation. Lastly, it is also widely involved in many real biological situations 
									which may link our experiments biologically to some real disease or human health case. The next question, however, comes to how 
									to maximize the chiral amplification power and how to quantitatively characterize/report such chiral amplification. To this end, 
									Dr. Li developed a multidimensional data visualization method with rational choice of individual coordinates/vectors, namely, the 
									collisional cross-sections (CCSs) for zero-, one- and two-copper-bound peptide species. These three CCSs are highly dependent on 
									the metal-binding events and thus the chiral effects can be evaluated. The next step was to plot each of these 3D vectors into a 
									3D scattering space. Furthermore, Dr. Li decided to use the spatial distance in this 3D scattering space to quantitatively characterize 
									the D/L Structural Difference (DLSD). The DLSD method has now been proven very useful in this type of evaluations. We also adopted the 
									same conception for Aβ oligomer chiral amplification and quantitative characterization. The authors named the whole set of this innovative 
									IM-MS-based analytical strategy as <strong>integrative chirality anatomy platform (iCAP</strong>, Figure 2).</P>
									<p>Looking into the near future, Dr. Li plans to extend the fragment-based in vitro study to full-length Aβ in vivo exploration, 
									starting with some in cell tests. Notably, we anticipate combining this iCAP conception with our previous D-amino acid residue 
									localization method (Anal. Chem. 2014, 86, 2972). This combined follow-up study can contribute to a systematic interrogation of 
									chiral inversion of various amino acid residues on full-length Aβ stereoisomers and allowing precise localization of the D-amino 
									acid residues. Ultimately, we hope to provide more concrete molecular basis for potentially new AD drug targets, including the 
									previously ignored structural differences caused by amino acid chirality. A bold idea is, if we found D-isomerized Aβ with significantly 
									decreased cytotoxicity, directed chiral therapy on Alzheimer’s disease patients through precise control of site-specific (e.g. Ser- and 
									Asp-residues) Aβ D-isomerization enzymes.</p>
									
									<h3><U><strong><font color='purple'>Future SMS: Targeted Proteomics-guided Native SMS</font></strong></U></h3>									
									<p>Next-generation Native SMS should smartly and comprehensively integrate the individual benefits and advantages of ambient MS (in situ MS),
									traditional native IM-MS and targeted proteomics. We are dedicated to pushing native IM-MS more and <strong>more "NATIVE"</strong>, with special focus on 
									technical improvements on analytical speed, sensitivity and throughput. These fundamental MS research should be indispensable prerequisite 
									of reporting <strong>more "real"</strong> and closer to biological circumstances, thus facilitating MS to be developed as something really meaningful to human
									disease fighting through protein-level drug-linked and receptor-related fundamental research and industrial applications.</p>
								</article>

						</div>
						
					</div>
					<div class='major-header' class="col-12">
						<ul class="actions">
							<li><a href="index.html" class="button icon solid fa-file"> Get Me Back Home </a></li>
						</ul>
					</div>
				</section>
				

			<!-- Footer -->
				<section id="footer">
					<div class="container">
						<header>
							<h2>Opening Positions or Collaborations? <strong>One-click Touch:</strong></h2>
						</header>
						<div class="row">
							<div class="col-6 col-12-medium">
								<section>
									<form method="post" action="#">
										<div class="row gtr-50">
											<div class="col-6 col-12-small">
												<input name="name" placeholder="Name" type="text" />
											</div>
											<div class="col-6 col-12-small">
												<input name="email" placeholder="Email" type="text" />
											</div>
											<div class="col-12">
												<textarea name="message" placeholder="Message"></textarea>
											</div>
											<div class="col-12">
												<a href="mailto:ligongyu1@126.com?subject=limslab-contact" class="form-button-submit button icon solid fa-envelope">Send Message</a>
											</div>
										</div>
									</form>
								</section>
							</div>
							<div class="col-6 col-12-medium">
								<section>
									<p>The <strong>LimsLab @ NKU</strong> (Nankai University) is always <strong>open</strong> to collaborate with multidiciplinary researchers
									and has multiple <strong>opening</strong> positions for personnel with all-level research backgrounds, including Postdoc, PhD and 
									undergruadate students. Please contact PI (<a href="people.html"><strong>Dr. Gongyu Li</strong></a>) for more information as listed below: </p>
									<div class="row">
										<div class="col-6 col-12-small">
											<ul class="icons">
												<li class="icon solid fa-home">
													Research Center for Analytical Science, Nankai University<br />
													94 Weijin Road, Tianjin 30071<br />
												</li>
												<li class="icon solid fa-phone">
													(000) 000-0000
												</li>
												<li class="icon solid fa-envelope">
													<a href="mailto:ligongyu1@126.com?subject=limslab-contact">ligongyu1@126.com</a>
												</li>
											</ul>
										</div>
										<div class="col-6 col-12-small">
											<ul class="icons">
												<li class="icon brands fa-twitter">
													<a href="https://twitter.com/gongyu_li">twitter@gongyu_li</a>
												</li>
												<li class="icon brands fa-instagram">
													<a href="https://scholar.google.com/citations?hl=en&user=slw69f0AAAAJ&view_op=list_works&sortby=pubdate">scholar.google.com/gongyu_li</a>
												</li>
												<li class="icon brands fa-dribbble">
													<a href="https://www.researchgate.net/profile/Gongyu_Li">researchgate.net/gongyu_li</a>
												</li>
												<li class="icon brands fa-facebook-f">
													<a href="https://www.facebook.com/gongyu.li.96/">facebook.com/gongyu_li</a>
												</li>
											</ul>
										</div>
									</div>
								</section>
							</div>
						</div>
					</div>
					<div id="copyright" class="container">
						<ul class="links">
							<li>&copy; LimsLab @ Nankai University.     All rights reserved.</li>
						</ul>
					</div>
				</section>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>